src/ImageRunModal.jsx

```diff
@@ -516,7 +516,7 @@ export class ImageRunModal extends React.Component {
         }
 
         // Strip out all non-allowed container image characters when filtering.
-        let regexString = searchText.replace(/[^\w_.:-]/g, "");
+        let regexString = searchText.replace(/[^/\w_.:-]/g, "");
```

Comment:
So this strips out the `/` right? But that is also handled below?

Reply:
eg: without this fix when you search for `docker.io/library/fedora` the final input regex string ends up being `/docker.iolibraryfedora/i` which when filtering podman search results matches nothing:
![image](https://github.com/user-attachments/assets/d0b3f51b-e56d-4888-8b1b-2b8ae36f5797)

with this fix the final regex is `/\/library\/fedora/i` so it can match viable images: 
![image](https://github.com/user-attachments/assets/0545a2f3-a97e-478f-b8b1-30e9ecbcecfb)

Reply:
Aha, silly me :)